code/style.css

```diff
@@ -1,4 +1,253 @@
+      color: white;
+    }
+    
+  a:hover {
+      color: white;
+      text-decoration: underline;
+    }
+    
+  .logo {
+    width: 100px;
+    height: 100px;
+    margin: 20px;
+    }
+
+    /*** BIG NEWS ***/
+
+  .big-news {
+      padding: 30px;
+      margin-top: 30px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+    }
+
+    .big-news-text {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+    }
+  
+    /*** NEWS GRID ***/
+
+
+    .other-news {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+      
+    }
+      
+    .news1, .news2, .news3, .news4, .news5, .news6, .news7, .news8 {
+      display:flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+      background-color: #00334d;
+      margin: 10px;
+    }
+  
+}
+
+
+/*** TABLET RESPONSIVE ***/
+
+@media (min-width: 668px) and (max-width: 1024px) {
+  
+  header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+    
+  nav {
+    width: 50%;
+    margin: 20px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .order-1{
+    order: 1;
+  }
+    
+  .order-2 {
+    order: 2;
+  }
+  
+  .order-3 {
+    order: 3;
+  }
+  
+  .order-4 {
+    order: 4;
+  }
+    
+  a:link {
+    text-decoration: none;
+    color: white;
+  }
+    
+  a:visited {
+    text-decoration: none;
+    color: white;
+  }
+    
+  a:hover {
+    color: white;
+    text-decoration: underline;
+  }
+    
+  .logo {
+    width: 100px;
+    height: 100px;
+    margin: 20px;
+  }  
+  
+  .big-news {
+    padding-bottom: 20px;
+    margin-top: 30px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+  }
+
+  .big-news-text {
+    display: block;
+    align-items: flex-start;
+    padding: 20px;
+  }
+
+  .other-news {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-gap: 10px;
+  }
+      
+  .news1, .news2, .news3, .news4, .news5, .news6, .news7, .news8 {
+  grid-column: span 2;  
+  background-color: #00334d;
+  text-align: center;
+  }
+
+}
+
+
+/*** DESKTOP RESPONSIVE ***/
+
+@media (min-width: 1025px) {
+
+  header{
+    background-color: #002233;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    }
+    
+    nav {
+      width: 50%;
+      margin: 20px;
+      display: flex;
+      justify-content: space-between;
+    }
+    
+    a:link {
+      text-decoration: none;
+      color: white;
+      }
+    
+    a:visited {
+      text-decoration: none;
+      color: white;
+    }
+    
+    a:hover {
+      color: white;
+      text-decoration: underline;
+    }
+    
+    .logo {
+      width: 100px;
+      height: 100px;
+      margin: 20px;
+    }
+
+    .big-news {
```

Comment:
The image inside big-news becomes very big on a wide desktop. Consider adding something like:
```
.big-news img {
  max-width: 700px;
}
```